bottlerocket/testsys/src/testsys_types.rs

```diff
@@ -0,0 +1,88 @@
+use crate::error::{self, Result};
+use bottlerocket_types::agent_config::{ResultFormat, SonobuoyMode};
+use snafu::ResultExt;
+use std::{fs::read_to_string, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt, Clone)]
+pub(crate) struct SonobuoyPluginConfig {
```

Comment:
Maybe it's better not to use the same name for these two structs.

```suggestion
pub(crate) struct SonobuoyPluginArgs {
```